package.json

```diff
@@ -71,17 +71,17 @@
     "eslint-plugin-standard": "~4.0.1",
     "gulp": "~4.0.2",
     "gulp-cli": "~2.2.0",
-    "gulp-sass": "~4.0.2",
+    "gulp-sass": "^5.0.0",
```

Comment:
Might be better to use ~ instead of ^ because sometimes semver isnt followed properly and a breaking change can be introduced.